src/cli/src/error.rs

```diff
@@ -65,15 +41,47 @@ pub struct HttpError {
 }
 
 impl From<http_types::Error> for CliError {
-    fn from(e: http_types::Error) -> Self {
-        CliError::HttpError {
-            source: HttpError { inner: e },
-        }
+    fn from(inner: http_types::Error) -> Self {
+        Self::HttpError(HttpError { inner })
     }
 }
 
 impl CliError {
     pub fn invalid_arg<M: Into<String>>(reason: M) -> Self {
         Self::InvalidArg(reason.into())
     }
+
+    pub fn into_report(self) -> color_eyre::Report {
+        use color_eyre::{Report, Section};
+        use CliError::*;
+        use TheClusterError::*;
+        use TheCheckError::*;
+
+        match self {
+            e
```

Comment:
typos?

Reply:
Nope, that's an unfortunate side-effect of `cargo fmt` in this case because the match expression is so long. It originally looked like

```rust
match self {
    e @ ClusterError(...) => {...}
}
```

Which is a way of preserving the original value instead of picking out a destructured subpiece. Essentially `e` is equivalent to `self` in this case. So actually, thinking about it, we could probably do without the `e @`.

But it is used right below in `let report = Report::from(e)`. If we wanted to get rid of `e @`, we would replace that with `let report = Report::from(self)`

Reply:
can you change from `e` to something else.  That way, it reduce confusion and increase readability